README.md

```diff
@@ -21,11 +21,15 @@ Partner images are VM images which are created by third parties which GitHub Hos
 
 | Image | YAML Label | Included Software | Rollout Progress of Latest Image Release |
 | --------------------|---------------------|--------------------|---------------------|
-| [ARM-Partner-AVH](https://resources.github.com/arm-gihub-actions-beta/) | custom in larger runners| [AVH Inventory](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh_inventory.html ) | In Beta
+| [Ubuntu 22.04 by Arm Limited](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/) | custom in larger runners| [Ubuntu 22.04 Inventory](/images/arm-ubuntu-22-image.md) | In Beta (Private)
+| [ARM-Partner-AVH](https://resources.github.com/arm-gihub-actions-beta/) | custom in larger runners| [AVH Inventory](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh_inventory.html ) | In Beta (Private)
```

Comment:
nit: I recommend adding column `OS / Architecture`:

I will be useful when we add more partner images in future.

| OS / Architecture | Image | YAML Label | Included Software | Rollout Progress of Latest Image Release |
| - | --------------------|---------------------|--------------------|---------------------|
| Linux, ARM64 | [Ubuntu 22.04 by Arm Limited](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/) | custom in larger runners| [Ubuntu 22.04 Inventory](/images/arm-ubuntu-22-image.md) | In Beta (Private)
| Linux, x64 | [ARM-Partner-AVH](https://resources.github.com/arm-gihub-actions-beta/) | custom in larger runners| [AVH Inventory](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh_inventory.html ) | In Beta (Private)